api-docs/api-reference/methods/post-create-scheduled-backup-with-copy.rst

```diff
@@ -0,0 +1,143 @@
+
```

Comment:
Uugh, the documentation inpost-create-backup-version-accountid-backups.rst has a table and text descriptions for all the parameters and behaviors.
Now, I'm back to combining the content in the new rsts in this PR into the existing rst files since the amount of duplicated information is not insignificant.
The API docs do not have to be written to highlight new functionality, that is the responsibility of the release notes and the release notes should probably be updated to refer to the api docs for details.